### ISSUE TITLE
About the accuracy

### ISSUE THREAD

**Issue by 11510880** [leocances/ESC50#1]
Why the accuracy is only 2%？

**Comment by leocances** [comment]
Hi,

Thank for your attention

It is because I wrote a terribly bad model to perform the classification!
I open that project when I first start to learn about Deep learning, Keras, and Tensorflow and then I forgot about it. Now I should be able to do something that actually works.

Thank for the reminder, I will update the code soon
